public/js/user.js

```diff
@@ -318,6 +323,28 @@ function saveMarker(e) {
   return false;
 }
 
+function updateMarker(id) {
+    var json = JSON.stringify({ });
+    $.ajax({
+      url: "/marker/"+id,
+      type: "PATCH",
+      data: json,
+      dataType: "text",
```

Comment:
I believe this should be:
```node
dataType: "json"
```
since you're expecting to get json back.[[ref](https://api.jquery.com/jquery.ajax/#jQuery-ajax-settings)] Note I also removed the trailing comma that isn't needed.

Reply:
@bmamlin My bad, updated the PR :)